README.md

```diff
@@ -114,6 +110,56 @@ All the following attributes correspond to the ones listed in the
 * [on-upload-complete](https://uploadcare.com/docs/api_reference/javascript/widget/#widget-on-upload-complete)
 * [on-change](https://uploadcare.com/docs/api_reference/javascript/widget/#widget-on-change)
 
+Events usage example:
+
+```javascript
+  onUpload(info) {
+    console.log('fired Event "onUpload"');
+    console.log(info);
+  }
+
+  onChange(file) {
+    if(!file) {
+      return;
+    }
+    console.log('fired Event "onChange"');
+  // input file parameter depends on multiple-files widget attribute
+    if(this.multipleFiles) {
+  //  file contains 2 methods:
+  //  .promise() - returns the general promise for all uploadin files which resolves with the group info of uploaded files
+  //  .files() - returns the array of promises for each uploading file. Each promise resolves with uploaded file info
```

Comment:
an array of promises: one per each uploaded file?
resolves into? an uploaded file info